2020-02-18 00:38:16,728 INFO: Blog startup [in C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\__init__.py:43
2020-02-18 00:41:19,339 INFO: Blog startup [in C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\__init__.py:43 
2020-02-19 17:33:32,028 INFO: Blog startup [in C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\__init__.py:45
2020-02-19 20:17:28,963 INFO: Blog startup [in C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\__init__.py:45
2020-02-19 20:35:16,224 INFO: Blog startup [in C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\__init__.py:45
2020-02-19 20:35:29,965 ERROR: Exception on /login [GET] [in c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py:1890
Traceback (most recent call last):
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\routes.py", line 53, in login
    return render_template('login.html', title='Sign In', form=form)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\templates\login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\templates\base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
  File "C:\Users\ijh19\OneDrive\Documents\Projects\blog\app\templates\login.html", line 27, in block "content"
    <a href="{{ url_for('password_reset') }}">Click to Reset!</a>
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\flask\helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "c:\users\ijh19\onedrive\documents\projects\blog\env\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'password_reset'. Did you mean 'reset_password' instead?
